tests/ui/test_discopane.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from pages.desktop.discopane import DiscoveryPane, DiscoveryPanePage, AboutAddons
+
+@pytest.fixture
+def firefox_profile(base_url, firefox_profile, discovery_pane_url):
+    firefox_profile.set_preference('extensions.webapi.testing', True)
+    firefox_profile.set_preference('extensions.webservice.discoverURL', discovery_pane_url)
+    firefox_profile.update_preferences()
+    return firefox_profile
+
+@pytest.fixture
+def discovery_pane(selenium, discovery_pane_url):
+    return DiscoveryPanePage(selenium, discovery_pane_url).open().discovery_pane
+
+@pytest.mark.nondestructive
+def test_that_discovery_pane_loads(base_url, selenium, session_capabilities, discovery_pane):
+    assert discovery_pane.is_discopane_visible
+    assert (len(discovery_pane.uninstalled_addons) == 5)
```

Comment:
The outer brackets are not needed. This appears to be failing. It might be better to have a region for each add-on, returned via `discovery_pane.addons`. Each region can then have an `installed` property, which we could use to check that no add-ons are currently installed. This will allow for greater flexibility of use for these regions.